### ISSUE TITLE
.gitjournalignore?

### ISSUE THREAD

**Issue by contributor** [GitJournal/GitJournal#880]
Is there a way to exclude a file/note from displaying in list view?

It would be perfect if the app support .gitjournalignore (similar to git .gitignore and syncthing .stignore).

I used previously syncthing to sync my files to Android, which syncs only files that are not excluded in .stignore
After switching to GitJournal, the files are received directly from git, but there is no way to not display not needed files

**Comment by Wint3rmute** [comment]
The "Note File types" setting could be a solution to your problem:

![Screenshot_20221124_093255_GitJournal](https://user-images.githubusercontent.com/19196352/203732356-6d15531c-d3f1-45b9-82fa-19e715dacefb.jpg)
![Screenshot_20221124_093303_GitJournal](https://user-images.githubusercontent.com/19196352/203732378-00fb87eb-d312-4828-a63a-185f0d4586a5.jpg)

Its only allows for filtering based on the file extension though

**Comment by contributor** [comment]
@Wint3rmute Thank you, good to know, it can help in some cases!
Still, more flexible solution would be a welcome addition. Another benefit of .gitjournalignore - it is per repository setting, and backed up by git :)

**Comment by vHanda** [comment]
Agreed. This would be a good idea. Moving out of Issues as this is a Feature Request.
